packages/core/test/uikitExcludePattern_tests.js

```diff
@@ -46,6 +46,53 @@ tap.test(
     //act
     const result = uikitExcludePattern(pattern, uikit);
 
+    //assert
+    test.true(result);
+    test.end();
+  }
```

Comment:
Missing bracket leads to error while running tests

```suggestion
  }
);
```